test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -10494,6 +10505,16 @@ def start(self, args=None, launcher=None):
         :param launcher: Optional utility to invoke the launch of the service
         :type launcher: str or list
         """
+        if sched_name is not None:
+            cmd = [os.path.join(self.pbs_conf['PBS_EXEC'],
+                                'sbin', 'pbs_sched')]
+            cmd += [sched_name]
+            if sched_home is not None:
+                cmd += ['-d', sched_home]
+            ret = self.du.run_cmd(self.hostname, cmd, sudo=True, logerr=False,
+                                  level=logging.INFOCLI)
+            self.server.manager(MGR_CMD_LIST, SCHED)
+
```

Comment:
@visheshh ,
please catch the exception here.
Also you are not returning the "ret" value. Please do the same as is done in the else part of this method.

Why do you have line(10516):
self.server.manager(MGR_CMD_LIST, SCHED)

As far as I understand this way of launching multi-scheds is going to change in next interface, if I am not wrong.
You may add the comment mentioning the same, here.

Reply:
@varunsonkar , Thanks for the review i will update it accordingly.
I am doing list sched to update schedulers dictionary , as sched_port and sched_priv and sched_log will be updated once the scheduler starts.